### DISCUSSION TITLE
[Remix] Deployment failure: No such file ... @remix-run/dev

### DISCUSSION THREAD

**Discussion by WesleyYue** [vercel/community#1671] [Help] [answered]
### Summary

A deployment that worked fine on CLI v28.16.6 is now failing on v28.16.11 with this error:

```
ENOENT: no such file or directory, symlink '/var/task/node_modules/@remix-run/dev' -> '/vercel/path0/packages/web/node_modules/@remix-run/dev'
```

**Comment by TooTallNate** [comment, marked as answer]
Can you share a deployment URL and/or open source repo that is having this issue?

**Reply by WesleyYue** [reply]
https://vercel.com/double-automation/double/9VMcHSCKfFa1Q9hUG6bkuyy3ucnf

I confirmed the latest version where it works is 28.16.10

**Reply by TooTallNate** [reply]
Ok thank you. I see the issue now. Will have a patch up shortly.

**Reply by TooTallNate** [reply]
Ok fix is [here](https://github.com/vercel/vercel/pull/9591).

Would you mind verifying by creating a deployment with environment variable `VERCEL_CLI_VERSION` set with value `https://vercel-git-fix-gh-1671.vercel.sh/tarballs/vercel.tgz`?
